lib/ansible/modules/lineinfile.py

```diff
@@ -161,13 +174,29 @@
     group: root
     mode: '0644'
 
+- name: Replace a localhost entry searching for a literal string to avoid escaping
+  lineinfile:
```

Comment:
Here's no FQCN (same below).

Reply:
Good catch. The first few already had it previously so I assumed that the entire document was up to date